src/rest.cpp

```diff
@@ -629,7 +639,8 @@ static const struct {
       {"/rest/block/notxdetails/", rest_block_notxdetails},
       {"/rest/block/", rest_block_extended},
       {"/rest/chaininfo", rest_chaininfo},
-      {"/rest/mempool/info", rest_mempool_info},
+      {"/rest/mempool/info", rest_mempool_info_basic},
+      {"/rest/mempool/info/with_fee_histogram", rest_mempool_info_with_fee_histogram},
```

Comment:
Can't we just start to use query parameters?

Reply:
Would eventually be better but not scope of this PR (following the current scheme).

Reply:
Still voting against query parameters. With REST the general preference seems to be to turn queries into URLs, and query parameters tend to be avoided because they look ugly and are hard to remember.

Reply:
I don't think there's a "standard" here but with REST usually the URL path identifies a resource, a collection of resources, or an action - the verb is also relevant. But parameters are usually set in the URL query, order independent and can be optional. I also think this is more flexible, for instance, you could support `...?verbose=true` in all endpoints (just an example).

Reply:
This must be before the above line (order is important) otherwise `rest_mempool_info_with_fee_histogram ` is never called. I can commit a test after the fix.